app/views/layouts/application.html.haml

```diff
@@ -31,7 +31,7 @@
     = stylesheet_link_tag '/inert.css', skip_pipeline: true, media: 'all', id: 'inert-style'
 
     - if Setting.custom_css.present?
-      = stylesheet_link_tag custom_css_path, host: request.host, media: 'all'
+      = stylesheet_link_tag custom_css_path(v: Setting.settings_version), host: request.host, media: 'all'
```

Comment:
Do we maybe just have updated_at for the actual CSS setting?

Reply:
We could, but I decided against because we would have to manage cache ourselves (or hit the database each time) while the whole settings are cached at once. Only the values get cached, not the metadata.